recipes/qt/6.x.x/conanfile.py

```diff
@@ -277,11 +277,11 @@ def validate(self):
             raise ConanInvalidConfiguration("alsa and pulseaudio are not supported (QTBUG-95116), please disable them.")
 
     def requirements(self):
-        self.requires("zlib/1.2.11")
+        self.requires("zlib/1.2.12")
         if self.options.openssl:
-            self.requires("openssl/1.1.1m")
+            self.requires("openssl/1.1.1n")
         if self.options.with_pcre2:
-            self.requires("pcre2/10.37")
+            self.requires("pcre2/10.39")
```

Comment:
```suggestion
            self.requires("pcre2/10.37")
```
[this](https://github.com/conan-io/conan-center-index/commit/912758d0b9a95965c62b79d96856511126700ccf#diff-a07f33d6e09514bd2cec71f0c02f6991f85961b406df0b566f8c687f19995804R194-R195) breaks the detection of pcre2 by qt: https://github.com/qt/qtbase/blob/v6.2.4/cmake/FindWrapSystemPCRE2.cmake#L27-L32